library/src/main/java/com/pokegoapi/util/PokeNames.java

```diff
@@ -41,4 +42,53 @@ public static String getDisplayName(int pokedexNr, Locale locale) {
 			}
 		return str;
 	}
+
+	/**
+	 * Returns translated Pokemon name from english locale.
+	 * @param engName pokemon english name
+	 * @param newLocale target locale
+	 * @return translated pokemon name
+	 */
+	public static String translateName(String engName, Locale newLocale) {
```

Comment:
Make the method throwable with MissingResourceException instead of catching the exception.